incubator/vault/values.yaml

```diff
@@ -9,6 +9,13 @@ image:
   tag:
   pullPolicy: IfNotPresent
 
+vaultExporter:
+  enabled: false
+  # repository: grapeshot/vault_exporter
```

Comment:
If these values are all commented out, won't it fail if somebody sets `valtExporter` to true?    if these are sensible working defaults then they should be left uncommented.

Reply:
Yes, you are right. I've just uncommented all the sensible defaults except the CA cert one.